src/expandable-block/block.scss

```
@@ -167,4 +167,35 @@
       margin-left: $iui-s;
     }
   }
+
+  &.iui-borderless {
+    @include themed {
+      background-color: transparent;
+      border: initial;
+    }
+
+    .iui-header {
+      @include themed {
+        background-color: transparent;
+        border: initial;
+      }
+
+      &:hover {
+        @include themed {
+          background-color: t(iui-color-background-4);
+        }
+      }
```

I think you will need to add a border-radius here to soften those sharp edges
```suggestion
        border-radius: $iui-border-radius;
      }
```

---

Property added